config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -247,3 +247,8 @@ presets:
 - labels:
     preset-e2e-scalability-presubmits: "true"
   env:
+  # Number of bytes of an additional nodes objects annotation in a kubemark
+  # cluster. The annotation label is added to make nodes objects sizes similar
+  # to regular cluster nodes.
+  - name: KUBEMARK_NODE_OBJECT_SIZE_BYTES
```

Comment:
Actually, let's set this in the kubemark-common preset

Reply:
Done. I updated the PR description accordingly.